src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,84 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  fillteredTodos: Todo[];
```

Comment:
The variable 'fillteredTodos' is misspelled. It should be 'filteredTodos' to correctly reflect its purpose of storing filtered todo items.